kayak_core/src/binding.rs

```diff
@@ -1 +1,29 @@
+use std::time::Instant;
+
 pub use flo_binding::{bind, notify, Binding, Bound, Changeable, MutableBound, Releasable};
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct Debouncer {
+    last_updated: Instant,
+    time: f32,
```

Comment:
Personally, I think `threshold` or something similar might better communicate its purpose at a glance.

Reply:
Yeah I ended up not needing it, but I will change it to threshold because that makes sense. 👍